### ISSUE TITLE
property names as values aren't reported

### ISSUE THREAD

**Issue by jd-solanki** [Jordan-Hall/stylelint-use-logical-spec#22]
Hi,

Thanks for this great plugin.

This works really great except it is not throwing warn/error for properties that are used as a value, I should better explain it via an example:

```scss
transition: padding-left 0.15s ease;
will-change: padding-left;
```

Above, two styles aren't reported to get fixed.

Regards.

**Comment by Jordan-Hall** [comment]
I will take a look at animation specs and requirements and make this change this week. Sorry for the delay been busy on nxext project

**Comment by sc-keithbauer** [comment]
@Jordan-Hall - just commenting to raise visibility on this PR if you have not noticed it.

**Comment by Jordan-Hall** [comment]
Amazing thank you. @sc-keithbauer  want to be added to repo and NPM to make changes? I like the work you've done.

**Comment by Jordan-Hall** [comment]
Merged and release

**Comment by sc-keithbauer** [comment]
@Jordan-Hall - that's fine with me. 
Hopefully we won't have to make more updates to this until browsers alter support in any way.
